edgebuilder-install.sh

```diff
@@ -468,28 +468,18 @@ install_cli_rpm()
 uninstall_server()
 {
     export DEBIAN_FRONTEND=noninteractive
+    # check if edgebuilder-server is currently installed
```

Comment:
we need to first bring down the server components and all our containers and images before uninstalling the packages